.github/workflows/release-ios.yml

```
@@ -0,0 +1,150 @@
+          git add languages/swift/Rusaint
+          git commit -m "release(ios): ${{ steps.current_version.outputs.result }}"
+          git push
+      - name: Create release
+        run: |
+          curl -L \
+            -X POST \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${{ secrets.IOS_GITHUB_TOKEN }}" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/EATSTEAK/rusaint-ios/releases \
+            -d '{"tag_name":"${{ steps.current_version.outputs.result }}","name":"${{ steps.current_version.outputs.result }}"}'
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)